micro-enterprise/README.md

```diff
@@ -0,0 +1,38 @@
+---
+sidebar: auto
+---
+
+# Micro Enterprise
+
+## About
+Micro-Enterprise is an initiative to Orchistrate a complete professional lifecycle for UofT students.
+
+Micro-Enterprise aims to provide consistant and guarenteed internship opportunities for all MCS students but this summer, we aim to pilot this with a small subset of students and development processes.
+
+## Benefits
++ Get an internship by just working for a minimum of 5 hours a week.
++ Get guarenteed internship letters upon successfully completing projects.
++ Learn the art of enterprise-grade software delivery.
++ We're aligning with more and more companies and MNCs everyday, to get this program recognised. With your consent get a chance to be hired by them upon successfully completing projects.
```

Comment:
How about: 

"You get the chance to be hired by our ever-updating list of companies and MNCs"

You already mention before that you are guaranteed internship letters upon successfully completing projects.

Reply:
Change has been incorporated with in the revised commit. Please review the latest commit.